drivers/hmis/app/graphql/types/base_access.rb

```diff
@@ -14,7 +14,7 @@ def self.build(node_class, class_name: nil, &block)
         field :id, ID, null: false
 
         def id
-          [object.respond_to?(:id) ? object.id : nil, current_user&.id].compact.join(':')
+          [object.respond_to?(:id) ? object.id : nil, current_user&.id].compact.first
```

Comment:
Maybe this to make sure we always have an ID even if there is no object (as in the case of root permissions)

```suggestion
          [object.respond_to?(:id) ? object.id : nil, current_user&.id].compact.first
          (object.respond_to?(:id) && object.id) || current_user&.id
```

Reply:
I think it will already do that, with the `.compact.first`? If the object does not have an ID, it will be the current_user's id.

Reply:
Ah, yep, you're right